ytfzf

```diff
@@ -1537,7 +1537,7 @@ preview_img () {
 	url=${line##*|}
 
 	#make sure all variables are set{{{
-	_correct_json=$(jq -r --arg url "$url" '[.[]|select(.url==$url)]|unique_by(.ID)[0]' < "$video_json_file")
+	_correct_json=$(jq -nr --arg url "$url" '[inputs[]|select(.url==$url)][0]' < "$video_json_file")
```

Comment:
What exactly is this doing, I see from the manual that -n puts no input into the filter, and i see that `inputs` "outputs all remaining inputs, one by one". Which inputs exactly is it outputting.

In addition how does this fix the problem, it looks like it's doing the same thing?

Reply:
ahhh, thats really cool. I get it.